crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -2362,11 +2377,12 @@ fn test_multiple_conditions(
 ) {
     let val = condition as u8;
     let (a, conds) = cond_test(&format!(
-        "((({{h1}} ({{h2}} (1234 ((({val} (100 ) (({val} (503 ) (({val} (90 )))))"
+        "((({{h1}} ({{h2}} (1234 (((51 ({{h2}} ({} ) (({val} (100 ) (({val} (503 ) (({val} (90 )))))",
+        if condition == RESERVE_FEE { 541 } else { 1234 }
```

Comment:
This passed previously because we didn't create a coin, right? It's not obvious that making this change is simpler than to just not expect `ELIGIBLE_FOR_DEDUP` being set